app/helpers/green_lanes_helper.rb

```diff
@@ -69,9 +62,14 @@ def render_all_exemptions(assessments)
   end
 
   def category_assessments_checked(category_assessment_id)
-    exemptions = params[:exemptions]
-    return if exemptions.nil? || exemptions[:category_assessments_checked].nil?
+    params.dig(:exemptions, :category_assessments_checked, category_assessment_id.to_s)
+  end
+
+  def dig_category_answer(answers, category, category_assessment_id)
```

Comment:
maybe we don't need this method,
it's just a dig and it is called `dig_category_answer` :thinking: